app/src/main/AndroidManifest.xml

```diff
@@ -9,12 +9,18 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.Multimatum">
+
         <activity
             android:name=".MainSettingsActivity"
-            android:exported="false" />
+            android:exported="false"/>
+        <activity
+            android:name=".AccountActivity"
+            android:parentActivityName=".MainActivity"/>
```

Comment:
For other activity we explicit exported value as "false", however it is not explicit for the AccoutActivity. This is just about code coherence since the default value is "false" anyway